be/src/vec/olap/block_reader.cpp

```diff
@@ -280,6 +280,11 @@ Status BlockReader::_unique_key_next_block(Block* block, MemPool* mem_pool, Obje
     if (_filter_delete) {
         int delete_sign_idx = _reader_context.tablet_schema->field_index(DELETE_SIGN);
         DCHECK(delete_sign_idx > 0);
+        if (delete_sign_idx <= 0 || delete_sign_idx >= target_columns.size()) {
```

Comment:
delete line 282 ?

Reply:
maybe we can find some bug with asan use DCEHCK